cli/src/test/scala/com/lightbend/rp/reactivecli/runtime/kubernetes/DeploymentJsonTest.scala

```diff
@@ -55,6 +56,26 @@ object DeploymentJsonTest extends TestSuite {
   val tests = this{
     "json serialization" - {
       "deployment" - {
+        "deploymentType" - {
+          "Canary" - {
+            (Deployment.generate(annotations, "apps/v1beta2", imageName, Deployment.ImagePullPolicy.Never, noOfReplicas = 1, Map.empty, CanaryDeploymentType).get.payload.hcursor --\ "metadata" --\ "name")
+              .focus
+              .contains(jString("friendimpl-v3-2-1-snapshot"))
+          }
+
+          "BlueGreen" - {
+            (Service.generate(annotations, "v1", clusterIp = None, BlueGreenDeploymentType).get.get.payload.hcursor --\ "metadata" --\ "name")
+              .focus
+              .contains(jString("friendimpl-v3-2-1-snapshot"))
+          }
+
+          "Rolling" - {
+            (Service.generate(annotations, "v1", clusterIp = None, RollingDeploymentType).get.get.payload.hcursor --\ "metadata" --\ "name")
+              .focus
+              .contains(jString("friendimpl"))
```

Comment:
do you want a stronger constraint on this test?  contains sounds like it would match "friendimpl-v3-2-1-snapshot" incorrectly, if I'm reading this right

Reply:
That's `Option.contains` so it'll only be true on an exact match.

Reply:
d'oh!